api/drop_chunks.md

```diff
@@ -54,19 +56,46 @@ The `older_than` and `newer_than` parameters can be specified in two ways:
     `SMALLINT` / `INT` / `BIGINT`. The choice of timestamp or integer
     must follow the type of the hypertable's time column.
 
+The `created_before` and `created_after` parameters can be specified in two ways:
+
+*   **interval type:** The cut-off point is computed as `now() -
+    created_before` and similarly `now() - created_after`.  This uses
+    the chunk creation time for the filtering.
+
+*   **timestamp, date, or integer type:** The cut-off point is
+    explicitly given as a `TIMESTAMP` / `TIMESTAMPTZ` / `DATE` or as a
+    `SMALLINT` / `INT` / `BIGINT`. The choice of integer value
+    must follow the type of the hypertable's partitioning column. Otherwise
+    the chunk creation time is used for the filtering.
+
 <Highlight type="warning">
 When using just an interval type, the function assumes that
 you are removing things _in the past_. If you want to remove data
 in the future, for example to delete erroneous entries, use a timestamp.
 </Highlight>
 
-When both arguments are used, the function returns the intersection of the
-resulting two ranges. For example, specifying `newer_than => 4 months` and
-`older_than => 3 months` drops all full chunks that are between 3 and 4 months
-old. Similarly, specifying `newer_than => '2017-01-01'` and
-`older_than => '2017-02-01'` drops all full chunks between '2017-01-01' and
-'2017-02-01'. Specifying parameters that do not result in an overlapping
-intersection between two ranges results in an error.
+When both `older_than` and `newer_than` arguments are used, the
+function returns the intersection of the resulting two ranges. For
+example, specifying `newer_than => 4 months` and `older_than => 3
+months` drops all full chunks that are between 3 and 4 months old.
+Similarly, specifying `newer_than => '2017-01-01'` and `older_than
+=> '2017-02-01'` drops all full chunks between '2017-01-01' and
+'2017-02-01'. Specifying parameters that do not result in an
+overlapping intersection between two ranges results in an error.
```

Comment:
Not clear what "full chunks" mean, so better to just omit it.

```suggestion
When both `older_than` and `newer_than` arguments are used, the
function returns the intersection of the resulting two ranges. For
example, specifying `newer_than => 4 months` and `older_than => 3
months` drops all chunks between 3 and 4 months old.
Similarly, specifying `newer_than => '2017-01-01'` and `older_than
=> '2017-02-01'` drops all chunks between 2017-01-01 and
2017-02-01. Specifying parameters that do not result in an
overlapping intersection between two ranges results in an error.
```

Reply:
done